### DISCUSSION TITLE
"BREAKING CHANGE error"

### DISCUSSION THREAD

**Discussion by JoaquinBN** [Web3Auth/Web3Auth#1345] [Help]
I have encountered myself with a whole new error to which I cannot find a viable solution. I am implementing an application on Ionic using VueJS framework. I've tried configuring the Webpack with a stream and crypto fallbacks. This solution allows me to compile the project without errors, but prevents me from accessing the Web3Auth integrated login. What can this be due to? I need help ASAP. Thank you very much !

![Screenshot 2023-02-19 171953](https://user-images.githubusercontent.com/47780609/219960695-0f0926d0-6b33-45d6-841b-0e605f53add1.jpg)

**Comment by CaptainUnknown** [comment]
According to the docs, you'd need to create webpack config the app directory named as config-overrides.js. The file should have the following config:

```js
const webpack = require("webpack");

module.exports = function override(config) {
  const fallback = config.resolve.fallback || {};
  Object.assign(fallback, {
    crypto: require.resolve("crypto-browserify"),
    stream: require.resolve("stream-browserify"),
    assert: require.resolve("assert"),
    http: require.resolve("stream-http"),
    https: require.resolve("https-browserify"),
    os: require.resolve("os-browserify"),
    url: require.resolve("url"),
  });
  config.resolve.fallback = fallback;
  config.plugins = (config.plugins || []).concat([
    new webpack.ProvidePlugin({
      process: "process/browser",
      Buffer: ["buffer", "Buffer"],
    }),
  ]);
  config.ignoreWarnings = [/Failed to parse source map/];
  config.module.rules.push({
    test: /\.(js|mjs|jsx)$/,
    enforce: "pre",
    loader: require.resolve("source-map-loader"),
    resolve: {
      fullySpecified: false,
    },
  });
  return config;
};
```

You can check out their [Integration Builder](https://web3auth.io/docs/integration-builder) to help you get started with the setup.

**Reply by JoaquinBN** [reply]
Oh great, thanks ! I have checked the integration builder for the vue platform and the vue.config.js file works perfectly. Thank you again for the fast answer.
